DesktopEdge/MainWindow.xaml.cs

```diff
@@ -440,8 +439,8 @@ public MainWindow() {
             notifyIcon.MouseClick += NotifyIcon_MouseClick;
             notifyIcon.ContextMenu = this.contextMenu;
 
-            IdentityMenu.OnDetach += OnDetach;
-            MainMenu.OnDetach += OnDetach;
+            IdentityMenu.HandleAttachment += HandleAttachment;
```

Comment:
The function for window attachment now handles detachment all-in-one.  As a result, renaming the function clarifies its duty to the program.